e2e/tests/stablex_test.rs

```diff
@@ -69,15 +69,25 @@ fn test_with_ganache() {
         )
         .from(Account::Local(accounts[1], None))
         .wait_and_expect("Cannot place first order");
-    close_auction(&web3, &instance);
 
+    let batch_id = instance.get_current_batch_id().call().wait().unwrap();
+    close_auction(&web3, &instance);
+    let wait_time = Duration::from_secs(310);
```

Comment:
While this is fine for now, I still think it's a bummer that the ganache e2e test now takes > 5 minutes in the worst case. 

Is there a reason why you want to do the sleeps in "one go" instead of waking up every e.g. 5s and re-determining action? This would allow us to build this fake System clock based on the ganache

Reply:
If I understand that approach correctly then we would have to introduce SystemClock trait, a new type that implements it based on the real SystemClock and one on Ganache (by sending it requests whenever the SystemTime is retrieved) and then command line options to switch.
To me that is too much extra logic that only exists for a test. If we had always been using the SystemClock and now wanted to create the Ganache test I don't think this would feel natural.
But if you think this is worth or I misunderstood the complexity then I can implement that. In that case let's move it into a separate PR after this one.

Reply:
I also agree this should be a separate PR.